base/reducedim.jl

```diff
@@ -87,7 +94,7 @@ end
 
 # initialization when computing minima and maxima requires a little care
 for (f1, f2, initval, typeextreme) in ((:min, :max, :Inf, :typemax), (:max, :min, :(-Inf), :typemin))
-    @eval function reducedim_init(f, op::typeof($f1), A::AbstractArray, region)
+    @eval function reducedim_init(f, op::typeof($f1), A::AbstractArrayOrBroadcasted, region)
```

Comment:
This change seems incomplete as `view` in L106 has no `Broadcasted` support.
we might need
```julia
reduced_view(A::AbstractArray, ri) = view(A, ri...)
reduced_view(bc::Broadcasted, ri) = Broadcasted(bc.style, bc.f, bc.args, ri)
```